rln-research/2022-07-22-rln-relay-journey.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+name:  ""
+title: ""
+date: 2022-10-18 10:00:00
+author: sanaztaheri
+published: true
+permalink: /waku-rln-relay-journey
+categories: research
+summary: The current state of the research and development of the WAKU-RLN-RELAY project.
+image: 
+discuss: 
+_includes: [math]
+---
+
+In this post, we would like to shed light on the current state of the research and development of the WAKU-RLN-RELAY project as well as its future direction and the areas of work to which the readers of this post may wish to contribute.
+
+# WAKU-RLN-RELAY: Why and How
+
+**Why**: The WAKU-RLN-RELAY project was started more than two years ago to protect the open p2p messaging protocol of [WAKU-RELAY](https://rfc.vac.dev/spec/11/) against spam and DDoS attacks.
```

Comment:
Link back to previous post on RLN

Reply:
sure, done https://github.com/vacp2p/research/pull/143/commits/33d5b37ec63e1271cf2e6a384bc0342d92fb3c30